nixos/modules/installer/cd-dvd/installation-cd-graphical-kde.nix

```diff
@@ -13,9 +13,7 @@ with lib;
       enable = true;
       enableQt4Support = false;
     };
-
-    # Enable touchpad support for many laptops.
-    synaptics.enable = true;
+    libinput.enable = true;
```

Comment:
There's an issue here. The plasma5 module doesn't enable `libinput` so it will be inconsistent for users who install NixOS and use Plasma5.

Reply:
If anything that means `libinput` should be the default for `plasma5`. 
Not enabling `libinput` by default on our install media makes a lot of input devices feel strange.

Reply:
@adisbladis That is precisely what I meant.

Reply:
Agreed, I didn't realize this wasn't the Plasma or ISO default.

Reply:
did this #68233